test/validation-error-handling.test.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 
-const { test } = require('tap')
+const t = require('tap')
 const Joi = require('@hapi/joi')
 const Fastify = require('..')
+const test = t.test
```

Comment:
Why did you make this change? The destructure is equivalent.

Reply:
I'm not sure why I changed it to be honest, so I reverted the change